templates/extra/resources.html

```diff
@@ -0,0 +1,289 @@
+{% extends "base.html" %}
+{% load staticfiles %}
```

Comment:
This top section looks good, but I like to break apart the blocks by whitespace, and the extends and imports. I would prefer if it looks like this:
```html
{% extends "base.html" %}

{% load staticfiles %}

{% block extrastyle %}
    ...
{% endblock %}

{% block content %}
    ...
```